### PR TITLE
feature/sensor-alarm-triggering

### PR DESCRIPTION
**Pull request by paga16-hash**

### CODE REVIEW

File: alarm/src/consumer.ts [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+import { IntrusionRule } from 'domain/dist/domain/security-rule/core/IntrusionRule.js'
+import { Device } from 'domain/dist/domain/device/core/Device.js'
+import { SecurityRuleService } from 'domain/dist/application/security-rule/SecurityRuleService.js'
+import { SecurityRuleServiceImpl } from 'domain/dist/application/security-rule/impl/SecurityRuleServiceImpl.js'
+import { DeviceType } from 'domain/dist/domain/device/core/impl/enum/DeviceType.js'
+import { DeviceTypeConverter } from 'domain/dist/utils/DeviceTypeConverter.js'
+import { EnvironmentDataFactory } from 'domain/dist/domain/device/factories/EnvironmentDataFactory.js'
+import { EnvironmentDataFactoryImpl } from 'domain/dist/domain/device/factories/impl/EnvironmentDataFactoryImpl.js'
+import kafkaManager from './utils/KafkaManager.js'
+import { afterEach } from 'vitest'
```
Comment (Mala1180) [inline alarm/src/consumer.ts:21]: Is this a typo?

File: domain/test/security-rule/exceeding-rule.test.d.ts
Comment (Mala1180) [inline domain/test/security-rule/exceeding-rule.test.d.ts:1]: Are these d files necessary?

**Review by Mala1180 — APPROVED** [review]
Nice job!

File: domain/test/security-rule/exceeding-rule.test.d.ts
Comment (paga16-hash) [reply domain/test/security-rule/exceeding-rule.test.d.ts:1]: removed :D

**Pull request merged**